### PR TITLE
Add a flake8 command to etstool.py

### PR DESCRIPTION
**Pull request by corranwebster**
This PR also adds a setup.cfg which makes flake8 run cleanly.  There are
things in the clean list that are likely errors and need to be fixed.

This doesn't add flake8 to CI, that should be a separate PR.

### CODE REVIEW

File: etstool.py [hunk 1]
```diff
@@ -462,6 +464,30 @@ def test_all(edm):
...
+    '--strict/--not-strict',
+    default=False,
+    help="Use strict configuration for flake8 [default: --not-strict]",
+)
+def flake8(edm, runtime, toolkit, environment, strict):
+    """Run a flake8 check in a given environment."""
+    parameters = get_parameters(edm, runtime, toolkit, environment)
+    config = ""
+    if strict:
+        config = "--config=flake8_strict.cfg "
```
Comment (rahulporuri) [inline etstool.py:484]: I'm not sure what this file is. should this be `setup.cfg`?

Also, I don't fully understand what the point of "strict" vs "non-strict" options for flake8 is - don't we always want to use "strict"?

Comment (corranwebster) [reply etstool.py:484]: This is a copy-pasta from TraitsUI - the idea was strict is something like what we are aiming for (and so new code should pass strict), but there are a lot of issues that need fixing before old code could pass the strict version.

Comment (corranwebster) [reply etstool.py:484]: Obviously I'll fix one way or another.

**Review by rahulporuri — APPROVED** [review]
LGTM for now. We'll need to come back and make changes anyway as we slowly address the flake8 issues.

**Pull request merged**